.github/workflows/control.yml

```diff
@@ -17,6 +17,11 @@ on:
     branches: [ main]
     tags: ["v*"]
   pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - closed
```

Comment:
Is there a reason why the workflow runs, when the PR is closed?

Reply:
yes, so that the release label is read and interpret.